src/libclipper/include/clipper/logging.hpp

```diff
@@ -0,0 +1,172 @@
+#ifndef CLIPPER_LOGGING_HPP
+#define CLIPPER_LOGGING_HPP
+
+#include <string>
+#include <sstream>
+
+#include <spdlog/spdlog.h>
+
+namespace clipper {
+
+static constexpr uint LOGGING_FORMAT_LENGTH = 18;
+static constexpr uint MAX_TAG_LENGTH = 10;
+// Defines the logging format as [HH:MM:SS.mmm][<LOG_LEVEL_1_CHAR>] <Message>
+// Note: <Message> includes a formatted, user-defined tag (see get_formatted_tag())
+static const std::string LOGGING_FORMAT = "[%T.%e][%L] %v";
+static const std::string LOGGER_NAME = "clipper";
+
```

Comment:
Can you add static convenience functions for log_debug, log_info, etc. that wrap the call to `Logger::get().log_*`?